src/themes/playful.css

```
@@ -351,107 +768,200 @@
+  --wa-color-brand-fill-highlight: var(--wa-color-brand-40);
+  --wa-color-brand-border-subtle: var(--wa-color-brand-30);
+  --wa-color-brand-border-highlight: var(--wa-color-brand-40);
+  --wa-color-brand-text-on-spot: var(--wa-color-brand-95);
+  --wa-color-brand-text-on-fill: var(--wa-color-brand-95);
+  --wa-color-brand-text-on-surface: var(--wa-color-brand-70);
+
+  --wa-color-success-spot: var(--wa-color-green-80);
+  --wa-color-success-spot-darker: var(--wa-color-green-60);
+  --wa-color-success-fill-subtle: var(--wa-color-green-20);
+  --wa-color-success-fill-highlight: var(--wa-color-green-40);
+  --wa-color-success-border-subtle: var(--wa-color-green-30);
+  --wa-color-success-border-highlight: var(--wa-color-green-40);
+  --wa-color-success-text-on-spot: var(--wa-color-green-20);
+  --wa-color-success-text-on-fill: var(--wa-color-green-95);
+  --wa-color-success-text-on-surface: var(--wa-color-green-70);
+
+  --wa-color-warning-spot: var(--wa-color-yellow-80);
+  --wa-color-warning-spot-darker: var(--wa-color-yellow-60);
+  --wa-color-warning-fill-subtle: var(--wa-color-yellow-20);
+  --wa-color-warning-fill-highlight: var(--wa-color-yellow-40);
+  --wa-color-warning-border-subtle: var(--wa-color-yellow-30);
+  --wa-color-warning-border-highlight: var(--wa-color-yellow-40);
+  --wa-color-warning-text-on-spot: var(--wa-color-yellow-20);
+  --wa-color-warning-text-on-fill: var(--wa-color-yellow-95);
+  --wa-color-warning-text-on-surface: var(--wa-color-yellow-70);
+
+  --wa-color-danger-spot: var(--wa-color-red-60);
   --wa-color-danger-spot-darker: var(--wa-color-red-40);
-  --wa-color-danger-fill-subtle: var(--wa-color-red-10);
-  --wa-color-danger-fill-highlight: var(--wa-color-red-20);
-  --wa-color-danger-border-subtle: var(--wa-color-red-20);
-  --wa-color-danger-border-highlight: var(--wa-color-red-30);
-  --wa-color-danger-text-on-spot: white;
-  --wa-color-danger-text-on-fill: var(--wa-color-red-70);
-  --wa-color-danger-text-on-surface: var(--wa-color-red-60);
+  --wa-color-danger-fill-subtle: var(--wa-color-red-20);
+  --wa-color-danger-fill-highlight: var(--wa-color-red-40);
+  --wa-color-danger-border-subtle: var(--wa-color-red-30);
+  --wa-color-danger-border-highlight: var(--wa-color-red-40);
+  --wa-color-danger-text-on-spot: var(--wa-color-red-20);
+  --wa-color-danger-text-on-fill: var(--wa-color-red-95);
+  --wa-color-danger-text-on-surface: var(--wa-color-red-70);
+
+  --wa-color-neutral-spot: var(--wa-color-base-40);
+  --wa-color-neutral-spot-darker: var(--wa-color-base-70);
+  --wa-color-neutral-fill-subtle: var(--wa-color-base-40);
+  --wa-color-neutral-fill-highlight: var(--wa-color-base-30);
+  --wa-color-neutral-border-subtle: var(--wa-color-base-30);
+  --wa-color-neutral-border-highlight: var(--wa-color-base-70);
+  --wa-color-neutral-text-on-spot: var(--wa-color-base-20);
+  --wa-color-neutral-text-on-fill: var(--wa-color-base-95);
+  --wa-color-neutral-text-on-surface: var(--wa-color-base-70);
```

```suggestion
  --wa-color-neutral-spot: var(--wa-color-base-40);
  --wa-color-neutral-spot-darker: var(--wa-color-base-70);
  --wa-color-neutral-fill-subtle: var(--wa-color-base-20);
  --wa-color-neutral-fill-highlight: var(--wa-color-base-30);
  --wa-color-neutral-border-subtle: var(--wa-color-base-30);
  --wa-color-neutral-border-highlight: var(--wa-color-base-40);
  --wa-color-neutral-text-on-spot: var(--wa-color-base-20);
  --wa-color-neutral-text-on-fill: var(--wa-color-base-95);
  --wa-color-neutral-text-on-surface: var(--wa-color-base-70);
```
I think this brings us a bit closer to the mock-ups.